### PR TITLE
Add object filter helper

### PR DESCRIPTION
**Pull request by marioreggiori**
### Checklist

* [ ] added release notes to `Unreleased` section in [CHANGELOG.md](CHANGELOG.md), if user facing change

### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### CODE REVIEW

**Comment by LittleFox94** [comment]
@marioreggiori please review this

as it was my code in the first place, I made it fit for merging - at least tests are green (except the e2e tests) and CodeClimate is happy now. As you opened the PR, I cannot set you as reviewer, though x)

File: pkg/utils/object/filter/filter.go [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+		values.Set(field, fmt.Sprintf("%v", value))
+	}
+
+	return values
+}
+
+func parseFilterableTag(field reflect.StructField) (isFilterable bool, filterName string, option string) {
+	filterName = field.Name
+
+	tag, ok := field.Tag.Lookup("anxcloud")
```
[the remark this answers is not in the document]
Comment (LittleFox94) [reply pkg/utils/object/filter/filter.go:70]: It's our tag and the only use it currently has is for marking a field as `identifier`. You cannot have a field that is both Identifier and Filterable, as you `Get` that Object via Identifier directly

I'm certain this is a good use of that tag

[the remark this answers is not in the document]
Comment (marioreggiori) [reply pkg/utils/object/filter/filter.go:70]: Alight then LGTM

[the remark this answers is not in the document]
Comment (marioreggiori) [reply pkg/utils/object/filter/filter.go:70]: ... can't approve as the PR creator.

**Review by LittleFox94 — APPROVED** [review]
approved by @marioreggiori in a roundabout way - see comments

**Comment by LittleFox94** [comment]
waiting with merge until ENGSUP-6077 is hotfix-resolved

**Pull request merged**